docs/DependencyAnalysis.rst

```diff
@@ -91,7 +91,8 @@ External Dependencies
 =====================
 
 External dependencies, including imported Swift module files and Clang headers,
-are tracked using a special ``depends-external`` set. The Swift driver
+are tracked using a special ``depends-external`` set. These dependencies refer
+to definitions that are external to the module. The Swift driver
```

Comment:
No, this is still wrong. These dependencies do not refer to definitions at all. They refer to files (as mentioned in the previous sentence).

Reply:
I finally got it: we write out *filenames* not *type names* for these. Sorry it took me so long. Thank you for your patience!

Reply:
And thank you for reviewing so quickly!